tika-bundle/pom.xml

```diff
@@ -112,320 +112,320 @@
   </dependencies>
 
   <build>
-    <plugins>
```

Comment:
ACK @Zarana-Parekh please rebase tika-bundle/pom.xml against master branch as the formatting changes make the patch contribution unclear.